### DISCUSSION TITLE
Error exporting!

### DISCUSSION THREAD

**Discussion by xhosch** [nicktylah/gnucash-mobile#19] [Q&A]
Hello and good evening,
when I press the "Export" button after selecting my desired directory, I get the following message. "Error exporting!" What might be the reason for this please?
Android version: 10 QKQ1.90910.002 
gnucash mobile 1.08

Best regards
Holger
Germany
![Screenshot_2022-05-06-19-44-40-674_com nicktylah gnucash_mobile](https://user-images.githubusercontent.com/50357925/167187243-02f9f78c-861b-4430-b849-204e4f1198f7.jpg)

**Comment by janekmi** [comment]
A little late and not an actual solution. But it is good to keep issues linked together: https://github.com/nicktylah/gnucash-mobile/issues/17
